zabbix/README.md

```diff
@@ -1,74 +1,106 @@
+* Generate the Rudder package if you do not already have the .rpkg file :
   ```
   cd zabbix
   make
   ```
 
-  The .rpkg file will be generated in the current directory.
+  - The .rpkg file will be generated in the current directory.
 
-- Install the package on the Rudder server :
+* Install the package on the Rudder server :
   ```
   /opt/rudder/bin/rudder-pkg install-file <plugin>.rpkg
   ```
 
-- Edit the configuration file /opt/rudder/etc/zabbix.conf. This config file has to contain your Zabbix
+* Edit the configuration file /opt/rudder/etc/zabbix.conf. This config file has to contain your Zabbix
   server's webservice API URL and the credentials to access it.
   You also need to provide the Rudder API URL, and a token to access it, which you will need to issue : 
   see https://www.rudder-project.org/rudder-api-doc/#api-_-Authentication
 
-Usage
------
+== Usage
 
-Now that the plugin is installed, you can check if it is working by adding new Rudder nodes. They will be 
+* Now that the plugin is installed, you can check if it is working by adding new Rudder nodes. They will be 
 automatically added to your list of Zabbix hosts (and removed on deletion of the Rudder node).
 
-This is achieved by Rudder's post-node-acceptance/deletion hooks, which are calling the plugin with the appropriate
+* This is achieved by Rudder's post-node-acceptance/deletion hooks, which are calling the plugin with the appropriate
 options on such events. The plugin will then make calls to the APIs of the Rudder and Zabbix servers to handle the
 modifications.
 
-In order to configure the monitoring of the Rudder nodes, you can provide Zabbix with parameters (macros) associated to its
-hosts. The "Monitoring parameter" generic method achieves this.
+* In order to configure the monitoring of the Rudder nodes, you can provide Zabbix with parameters (macros) associated to its hosts. The "Monitoring parameter" generic method achieves this.
 
-The data will be sent hourly to the Zabbix server. This can be changed by editing the /etc/cron.d/zabbix-rudder file.
+```
+Make sure for adding monitoring parameter, the key field must be written in uppercase.
+```
+
+* The data will be sent hourly to the Zabbix server. This can be changed by editing the /etc/cron.d/zabbix-rudder file.
 This operation can be manually executed by running
+
+Adding|Removing hosts
 ```
 /opt/rudder/bin/zabbix-plugin.py update
 ```
+
+Adding hosts
+```
+/opt/rudder/bin/zabbix-plugin.py hook addHost <id>
+```
+
+Removing hosts
+```
+/opt/rudder/bin/zabbix-plugin.py hook rmHost <hostname>
+```
 on the Rudder server.
 
+* In order to configure the monitoring to the Rudder nodes, you can provide Zabbix with templates and parameters (macros) associated to Zabbix hosts and will be synchronized with zabbix using its API.
+
+
+* This operation can be mannually executed by running:
+```
+/opt/rudder/bin/zabbix-plugin.py apply-configuration
+```
+
 Authors
```

Comment:
You need to remove the end (starting from "Authors"), we don't want that displayed in Rudder manual.